tests/e2e/tests/pilot/pilot_test.go

```diff
@@ -99,9 +101,9 @@ func TestPilot(t *testing.T) {
 		config.Verbosity = 3
 	}
 
-	// Only run the tests if the user has defined the KUBECONFIG environment variable.
-	if config.KubeConfig == "" {
-		t.Skip("Env variable KUBECONFIG not set. Skipping tests")
+	// Only run the tests if the user has defined the KUBECONFIG environment variable or a cluster registry directory
```

Comment:
I assume this will go away when the second PR gets in (kube config will be needed in all cases). Maybe we can get rid of it earlier.